lib/player.rb

```diff
@@ -0,0 +1,60 @@
+class Player
+  attr_accessor :arr_player1, :arr_player2, :player1, :player2
+  def initialize
+    @player1 = 'Player 1'
+    @player2 = 'Player 2'
+    @arr_player2 = []
+    @arr_player1 = []
+    @winning_streaks =
+      [[1, 2, 3],
+       [4, 5, 6],
+       [7, 8, 9],
+       [1, 4, 7],
+       [2, 5, 8],
+       [3, 6, 9],
+       [1, 5, 9],
+       [3, 5, 7]]
+  end
```

Comment:
I'm confident you can make this long code into a simple and more elegant way with some loops